hazenlib/utils.py

```diff
@@ -274,12 +314,14 @@ def get_field_of_view(dcm: pydicom.Dataset):
 def get_image_orientation(iop):
     """
     From http://dicomiseasy.blogspot.com/2013/06/getting-oriented-using-image-plane.html
+
     Args:
-        iop:
+        iop (list): values of dcm.ImageOrientationPatient - list of float
 
     Returns:
-
+        str: Sagittal, Coronal or Transverse
     """
+    # TODO: check that ImageOrientationPatient field is always available (every mannufacturer and enhanced)
```

Comment:
As per DICOM standard, ImageOrientationPatient should be in all MR DICOMs: https://dicom.innolitics.com/ciods/mr-image/image-plane/00200037

It should also be in Enhanced MR DICOMs, but it's buried further into the structure: https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image-multi-frame-functional-groups/52009230/00209116/00200037

Reply:
There will be a separate dev branch for changing the way these DICOM fields are parsed, so that they can be correctly extracted from both enhanced and normal DICOMs from all manufacturers.